docs/LEARNING.md

```diff
@@ -0,0 +1,7 @@
+## Learning Rust
+
+* [The Rust Programming Language](http://doc.rust-lang.org/stable/book/) is a great resource for getting started with Rust as well as diving deeper into specific features of Rust.
+* [Rust by Example](http://rustbyexample.com) shows you examples of the most common things you will be writing in Rust.
+* [Rustlings](https://github.com/carols10cents/rustlings) "Small exercises to get you used to reading and writing Rust code."
+* [#rust-beginners](https://client00.chat.mibbit.com/?server=irc.mozilla.org&channel=%23rust-beginners) "an IRC channel that loves answering questions at any depth"
```

Comment:
I'll fix that. I grabbed that link from rust-lang's community page, and it hard-codes the client00 part.